src/app/pages/observation-list/observation-list.page.ts

```diff
@@ -167,11 +171,13 @@ export class ObservationListPage extends NgDestoryBase implements OnInit {
     }
 
     this.listSearch = this.searchRegistrationService.pagedSearch(
-      this.searchCriteriaWhenThisPageIsActiveAndViewTypeList$
+      this.searchCriteriaWhenThisPageIsActiveAndViewTypeList$,
```

Comment:
Igjen, det blir litt rart at man sender inn hvilken tab man er på til denne metoden. this.searchCriteriaWhenThisPageIsActiveAndViewTypeList$ tar jo allerede høyde for om siden er aktiv eller ikke osv. Hvorfor trengs det at man ser på tab i tillegg?